responses/configuration/set.go

```diff
@@ -60,17 +54,11 @@ func (sc *SetConfigResponse) UnmarshalJSON(data []byte) error {
 	}
 
 	//If verified, copy the fields
-	if test.Code != nil {
-		sc.Code = *test.Code
-	}
-
 	if test.Description != nil {
 		sc.Description = *test.Description
 	}
-
-	if test.ExpectedValues != nil {
-		sc.ExpectedValues = test.ExpectedValues
-	}
+	// The field "Success" is equally valid when it holds a "true" or "false" value, so verification is irrelevant.
+	sc.Success = test.Success
```

Comment:
Nitpick - for consistency with the struct layout, I'd move this assignment before the `if test.Description...` line above.